tests/unit/test_package.py

```diff
@@ -119,6 +119,12 @@ def test_deploy_with_fallback_location(self, mock_put, mock_sudo, mock_isfile):
                                     use_sudo=True,
                                     temp_dir='/tmp')
 
+    @patch('prestoadmin.package.os.path.isfile')
+    def test_deploy_invalid_local_path(self, mock_isfile):
+        mock_isfile.return_value = False
+        invalid_path = '/invalid/path'
+        self.assertRaisesRegexp(SystemExit, 'RPM file not found at %s' % invalid_path,  package.deploy, invalid_path)
```

Comment:
It looks like you have an extra space after the command and the `package.deploy` function.